services/dynamodb/src/main/java/software/amazon/awssdk/services/dynamodb/DynamoDbRetryPolicy.java

```diff
@@ -35,7 +35,7 @@
     /**
      * Default max retry count for DynamoDB client, when using the LEGACY retry mode.
      **/
-    private static final int LEGACY_MAX_ERROR_RETRY = 8;
+    private static final int LEGACY_MAX_ERROR_RETRY = 10;
```

Comment:
The max error retry for legacy mode in v2 should still be 8.

Reply:
Thanks...
Added new STANDARD_MAX_ERROR_RETRY and junit test cases.